app/api/dao/admin.py

```diff
@@ -21,21 +21,21 @@ def assign_new_user(assigner_user_id, data):
         new_admin_user_id = data['user_id']
 
         if assigner_user_id is new_admin_user_id:
-            return {"message": "You cannot assign yourself as an Admin."}, 403
+            return {"message": constants.USER_CANNOT_ASSIGN_HIMSELF_AS_ADMIN}, 403
```

Comment:
```suggestion
            return {"message": constants.USER_CANNOT_ASSIGN_SELF_AS_ADMIN}, 403
```
I'm not sure if this is a good correction, but avoid using pronouns in your code, pronouns that assume the gender of the user. I haven't check this on other places, but be aware of this. Do a search for pronouns in the rest of your commit.

Reply:
@isabelcosta I'll remove all the pronouns :+1: